src/App.js

```diff
@@ -7,22 +7,24 @@ class App extends Component {
     value: '',
   }
 
-  addTask = () => this.setState(() => !!this.state.value.trimRight() && !this.state.tasks.includes(this.state.value) ? this.state.tasks.push(this.state.value) : null)
+  addTask = () => this.setState(( prevState ) => Boolean(prevState.value.trimRight()) && !prevState.tasks.includes(prevState.value) ? prevState.tasks.push(prevState.value) : null)
 
-  removeTask = idToRemove => this.setState({tasks: this.state.tasks.filter((task, id) => id !== idToRemove )})
+  removeTask = idToRemove => this.setState(( prevState ) => ({tasks: prevState.tasks.filter((task, id) => id !== idToRemove )}))
 
+  onChangeInputHandler = event => this.setState({value: event.target.value})
   render() {
+    const { tasks } = this.state
```

Comment:
it would be nice to have a convention like -> always emtpy line before return keyword. It improves readability.